### PR TITLE
Add status bar icons configuration 

### LINKED ISSUE #8: [feature] choose what to display in status bar for each component

### PR DESCRIPTION
**Pull request by haohanyang**
fix #8

### CODE REVIEW

File: eul/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -135,3 +135,4 @@
 "ui.network" = "Сеть";
 "ui.menu_view" = "Вид Меню";
 "ui.empty" = "Пусто";
+"ui.status_bar_view" = "Status Bar View";
```
Comment (haohanyang) [inline eul/ru.lproj/Localizable.strings:138]: @JiLiZART need your help of translation

File: eul/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -135,3 +135,4 @@
 "ui.network" = "Netzwerk";
 "ui.menu_view" = "Menü Ansicht";
 "ui.empty" = "Empty";
+"ui.status_bar_view" = "Status Bar View";
```
Comment (haohanyang) [inline eul/de.lproj/Localizable.strings:138]: @akeschmidi need your help of translation

**Review by haohanyang — COMMENTED** [review]
Need some translations @JiLiZART @akeschmidi

File: eul/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -135,4 +135,4 @@
 "ui.network" = "Netzwerk";
 "ui.menu_view" = "Menü Ansicht";
 "ui.empty" = "Empty";
-"ui.status_bar_view" = "Status Bar View";
+"ui.status_bar_view" = "Status Bar Ansicht";
```
Comment (haohanyang) [inline eul/de.lproj/Localizable.strings:138]: @akeschmidi can you have a review of the translation?

File: eul/ru.lproj/Localizable.strings [hunk 4]
```diff
@@ -135,4 +135,4 @@
 "ui.network" = "Сеть";
 "ui.menu_view" = "Вид Меню";
 "ui.empty" = "Пусто";
-"ui.status_bar_view" = "Status Bar View";
+"ui.status_bar_view" = "Вид Статус Бар";
```
Comment (haohanyang) [inline eul/ru.lproj/Localizable.strings:138]: @JiLiZART  can you have a review of the translation?

File: eul/de.lproj/Localizable.strings (hunk 3 above)
Comment (akeschmidi) [reply eul/de.lproj/Localizable.strings:138]: that's correct

File: eul/ru.lproj/Localizable.strings (hunk 4 above)
Comment (JiLiZART) [reply eul/ru.lproj/Localizable.strings:138]: Вид Статус Бара

**Comment by haohanyang** [comment]
Translation calibration is finished

**Comment by gao-sun** [comment]
thank you for contributing! will take a look tonight

**Comment by gao-sun** [comment]
super busy today, will take a look by the end of this week

**Comment by gao-sun** [comment]
image

just tried but did not found the functional difference between "Components" tab and "Status Bar View" tab, did I miss anything?

**Comment by haohanyang** [comment]
So am I misunderstanding #8?

**Comment by gao-sun** [comment]
Sorry the description of #8 looks ambiguous. Actually I would like to provide a more precise control of display content. Say some user just wanna have CPU percentage in status bar instead of two rows. I think a general solution would be preferred, which covers component display and configuration.

**Comment by haohanyang** [comment]
Fine. I will close the PR and think of it

**Comment by gao-sun** [comment]
thank you for contributing anyway! we can discuss in issues first and move to next steps once aligned

**Comment by gao-sun** [comment]
i'll close this PR for housekeeping, feel free to let me know if you got more ideas

**Pull request closed without merging**